cdhweb/static_src/global/components/sidenav.scss

```diff
@@ -1,71 +1,24 @@
-/*
-  This component has a 3d box like the homepage hero. The difference is that
-  this version aligns the *box* to the grid, rather than the content.
-  It was too fiddly to make a mixin that supported both.
-  
-  TODO: This should be made into a mixin or something when we do other such boxes,
-  e.g. the feature streamfield block and the newsletter signup streamfield block.
-  Might be tricky as they seem to have different border sizing rules to the sidenav.
-*/
-
 .sidenav {
-  --box-3d-side-size: 18px;
-  --box-border-width: 6px;
+  --three-dee-side-size: 18px;
```

Comment:
I'm not just being a dick btw. Var names (like class names) aren't supposed to start with a number.

Reply:
`isometric` would be another potential colour for the bike shed. But this is fine.